### ISSUE TITLE
Add the possibility for folders and paging

### ISSUE THREAD

**Issue by jensgro** [usephoton/photon-11ty#25]
If I understand your description correct, this project collects all photos on one page. I would be interested in different collections.  And can there be a paging mechanism for a  larger amounts of photos?

**Comment by coolcut** [comment]
> If I understand your description correct, this project collects all photos on one page.
Yes, this is the current state.

> I would be interested in different collections.
Collection meaning grouping of different photos? Something like categories, where you add some photos to the category e.g. "London 2018" and a separate collection with this name and all the photos that are associated with this category appear?

> And can there be a paging mechanism for a larger amounts of photos?
This is definitely something worth considering.

**Comment by jensgro** [comment]
yes, collections as categories. the same you could do in photos from Google. Maybe this could  simply be solved with the collection from eleventy. But what about the actual central folder for adding new images? What about placing a folder inside this central upload-folder that is named like the respective category it should be displayed in?

I am just trying to imagine how I could use your tool with more than just a handfull of images.
